src/metabase/email/alert.hbs

```diff
@@ -34,7 +34,7 @@
             </table>
             {{{computed.alert_content}}}
             <p style="color: {{payload.style.color_text_medium}}; font-size: 12px; font-weight: 400">
-            {{computed.alert_condition}}
+            {{#with payload.alert}}{{#if (equals alert_condition "goal")}}{{#if alert_above_goal}}when this question meets its goal{{else}}when this question goes below its goal{{/if}}{{else}}whenever this question has any results{{/if}}{{/with}}
```

Comment:
This could benefit from some line breaks IMHO, won't the HTML take care of ignoring the extra whitespace?

Reply:
yea it does, tests were annoyed with it :D, added line breaks and fixed the test: https://github.com/metabase/metabase/pull/50004/commits/245c2012810215f3cbd7109a54bb37983e4b1e7f